Lib/plistlib.py

```diff
@@ -518,7 +518,7 @@ def _is_fmt_xml(header):
     prefixes = (b'<?xml', b'<plist')
 
     for pfx in prefixes:
-        if header.startswith(pfx):
+        if header.lstrip().startswith(pfx):
```

Comment:
Please add a test in `test_plistlib.py` that is something similar to https://github.com/python/cpython/blob/a1952122a3a20272e4db9e3c6205009a91c73ccf/Lib/test/test_plistlib.py#L481 but with valid data and with a space before " <plist" so that it should decode correctly. Also a similar test for xml tags is needed I guess.

Before PR :

```
>>> plistlib.loads(b" <plist><integer>1</integer></plist>")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/karthikeyansingaravelan/stuff/python/cpython/Lib/plistlib.py", line 1000, in loads
    return load(
  File "/Users/karthikeyansingaravelan/stuff/python/cpython/Lib/plistlib.py", line 986, in load
    raise InvalidFileException()
plistlib.InvalidFileException: Invalid file
```

After PR :

```
>>> plistlib.loads(b" <plist><integer>1</integer></plist>")
1
```

Reply:
On it!

Reply:
Added tests in 7cfebbb